wiki/Manual.md

```diff
@@ -676,10 +676,10 @@ If you encounter DDoS protection errors and cannot reach a particular exchange t
 - run your software in a distributed network of servers
 - ...
 
-# Markets
+# <a id="markets">Instruments & Markets</a>
```

Comment:
I would rather keep the initial title here, cause there are various links to it all over the place. If we do change titles here, we should also change links here: https://github.com/ccxt/ccxt/wiki (ccxt.wiki.git)

Reply:
The reason I've implemented the `<a id="markets">...` tag was to retain that and avoid breaking. So, the link will stay same nevertheless the title change. Am I wrong?

Reply:
I don't think it will work with the RTD, check this other comment: https://github.com/ccxt/ccxt/pull/10857#discussion_r767178164

Reply:
Ah I see. you mentioned `<a>` tags to break RTD.

Reply:
If so, then let's be title same as it was.